sink/test-sink

```diff
@@ -155,7 +155,7 @@ class TestGitHub(unittest.TestCase, ):
         # Should terminate by itself due to /expect
         ret = sink.wait()
         sink.stdin.close()
-        self.assertEqual(ret, -signal.SIGTERM)
+        self.assertEqual(ret, 1)
```

Comment:
...where do we test that we get `77`?

Reply:
we don't, the tests here don't test collisions and i'm not sure how